src/Struct/FilterStruct.php

```diff
@@ -19,12 +21,32 @@ class FilterStruct {
     public $value;
 
     public function __construct($fieldName, $operator, $value) {
-        $this->fieldName = $fieldName;
+        if (strpos($fieldName, self::$LOGICAL_OR)) {
+            $this->fieldName = explode(self::$LOGICAL_OR, $fieldName);
+        } else {
+            $this->fieldName = $fieldName;
+        }
         $this->operator = $this->transformOperator($operator);
         $this->value = $value;
     }
 
     public function apply($object) {
+        if (is_array($this->fieldName)) {
+            $me = $this;
+
+            $subquery = function ($query) use ($me) {
+                $query = $query->where($me->fieldName[0], $me->operator, $me->value);
+
+                for ($fieldIndex = 1; $fieldIndex < count($me->fieldName); $fieldIndex++) {
```

Comment:
For side note, this one can shorten to
```
$subquery = function ($query) {
    for ($fieldIndex = 0; $fieldIndex < count($this->fieldName); $fieldIndex++) {
```

But it's your call to include this code or not.